tests/test_sources.py

```diff
@@ -80,6 +82,25 @@ def test_yahoo_bars_to_panel_source(self):
             self.assertEqual(next(stocks_iter), event['sid'])
 
 
+class TestSqlSource(TestCase):
+    def test_sql_source(self):
+        source, df = factory.create_test_sql_source()
+        assert isinstance(source.start, pd.lib.Timestamp)
```

Comment:
Okay, I didn't realize we had this pattern still. If you don't want to update the older tests, we can still support the cleaner way going forward and change it here.